.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build geth
         run: |
-          cd cmd/geth && /opt/go/1.18.1/bin/go build
+          cd cmd/geth && /opt/go/1.19.5/bin/go build
```

Comment:
I think 1.19 is no longer supported.  At some point, we should update to 1.21.

Not a blocker.